src/plugins/Trader/UI/PriceChanged.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import classNames from 'classnames'
+import { useColorStyles } from '../../../utils/theme'
+import { makeStyles, Theme, createStyles } from '@material-ui/core'
+
+const CHAR_RISE = '\u25B2' // ▲
+const CHAR_FALL = '\u25BC' // ▼
```

Comment:
If this is rendered as emoji on some platform, it will create a in-consistency effect. I suggest to use SVG icons from mui library if possible.